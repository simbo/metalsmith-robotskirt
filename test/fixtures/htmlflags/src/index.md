
![a not rendered image](/some/image.png)

[a not rendered link](http://domain.tld/)

Can someone please tell me how to generate style tags with markdown, so i can
write a test for preventing it?  At least, this text is a test for hard line
breaks.
