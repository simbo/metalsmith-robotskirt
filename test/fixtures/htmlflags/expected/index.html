<p>![a not rendered image](/some/image.png)</p>

<p>[a not rendered link](http://domain.tld/)</p>

<p>Can someone please tell me how to generate style tags with markdown, so i can<br>
write a test for preventing it?  At least, this text is a test for hard line<br>
breaks.</p>
